includes/class-admin-settings.php

```diff
@@ -265,18 +293,20 @@ public function the_settings_page() {
 				'reset'       => 'reset',
 				'reset-nonce' => wp_create_nonce( 'aspireupdate-reset-nonce' ),
 			),
-			admin_url( 'index.php?page=aspireupdate-settings' )
+			network_admin_url( 'index.php?page=aspireupdate-settings' )
 		);
 		?>
 		<div class="wrap">
 			<h1><?php esc_html_e( 'AspireUpdate Settings', 'AspireUpdate' ); ?></h1>
-			<form id="aspireupdate-settings-form" method="post" action="options.php">
+			<form id="aspireupdate-settings-form" method="post" action="index.php?page=aspireupdate-settings">
 				<?php
 				settings_fields( $this->option_group );
 				do_settings_sections( 'aspireupdate-settings' );
 				?>
 				<p class="submit">
-					<input type="submit" name="submit" id="submit" class="button button-primary" value="Save Changes">
+					<?php wp_nonce_field( 'aspireupdate-settings' ); ?>
+					<?php submit_button( esc_html__( 'Save Changes', 'aspireupdate-settings' ), 'primary', 'submit', false ); ?>
+
 					<a href="<?php echo esc_url( $reset_url ); ?>" class="button button-secondary" >Reset</a>
```

Comment:
Same clarification request about if this is necessary for this issue or if it's aprt of another?

Reply:
It's necessary for sending a nonce with the submit button. I did change the link to use the `submit_button()` though it may not be technically necessary to do that.

Reply:
Works for me, thanks!